benchmarks/TerraFX.Interop.Mimalloc.AllocatorsBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <NoWarn>
+      $(NoWarn);
+
+      <!--Missing XML comments-->
+      CS1591
+    </NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="BenchmarkResultsInterpreted.md" />
+    <None Include="BenchmarkResultsRaw.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
```

Comment:
This should be just: `<PackageReference Include="BenchmarkDotNet" />`

You should then add `<PackageReference Update="BenchmarkDotNet" Version="0.13.1" />` in:
https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/Directory.Build.targets#L36-L41

Even better, you can centralize the benchmark reference in a `<repoRoot>/benchmarks/Directory.Build.props` like we do for tests here: <https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/tests/Directory.Build.props#L25-L29>